docs/partner_editable/regions.adoc

```diff
@@ -5,4 +5,5 @@
 // - ca-central-1 (Canda)
 // - ap-south-1 (Mumbai)
 
-This Quick Start can be deployed in all AWS Regions where the services described in <<architecture1>> are available.
\ No newline at end of file
+This Quick Start can be deployed in all AWS Regions where the services described in <<architecture1>> are available.
+//TODO Shivansh, Aren't deployment guides supposed to list the specific Regions now? As in "The following Regions are currently supported by this Quick Start..." 
```

Comment:
Happy to do. However, I thought we were waiting for the automation to do this. But, I think I will do it for now.

Reply:
👍